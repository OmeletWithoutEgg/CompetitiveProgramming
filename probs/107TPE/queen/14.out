12045828.39 76800677.93
