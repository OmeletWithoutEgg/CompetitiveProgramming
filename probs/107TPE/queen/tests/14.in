356
53467610 33185020
11030969 7080552
-72946679 15269744
-68683519 15356429
-82172926 82540059
11747297 12767569
53030993 -16726290
-97829327 47923372
81554924 29339068
-82164322 18295550
-59062779 81769680
-96147606 62965105
-52228475 25693988
-78538744 84294114
11062256 6753228
-46669785 36666169
-44573839 51833308
-70176620 85617300
-67814706 85355479
-60960016 17909034
22184894 24606036
73840133 -15048743
-97017804 41453873
-95671779 36336951
102133567 3927053
-93393678 30835668
91277732 -8374550
-66660381 85124963
-85658003 80041219
11208433 10695472
-86231416 21300742
-44570907 51733333
101555422 2399310
102714308 6574207
92450680 24062506
51198780 -16621619
98426334 18899461
34925557 -13762402
10999457 7557797
-73387835 15312297
15280435 18794506
20357098 23378588
-97839347 52723589
-72558862 85600375
39034247 -14842681
-58584986 81438305
-55241964 78620226
-84298857 81122190
30136388 -12122975
-65667782 84871613
-69254282 15292454
100396278 16235972
-77480306 16178836
12658143 1520029
19123700 22451390
31634835 29124613
13786700 16833579
12161007 2586286
91717458 -8099707
-97132426 42050047
13271905 414689
11028438 9328492
-88204597 23256573
95952068 21385296
-51089395 73521308
100149896 -183223
57895339 33249109
21599254 24229761
101933073 3340435
100869918 1012851
51268930 33066123
-97372776 43474819
14853224 18279543
19536972 -6332362
14804788 18219141
25460973 -10026636
19659735 22865361
-79883293 83736629
-87848262 77965879
101111401 1466336
21843276 24388376
11692792 12603637
-97841286 48156537
-44792692 55138191
14909572 -1909521
-96712635 60797842
-64771857 16242537
-60920729 17930715
83942855 -12011706
-94782880 33904707
-44642439 53312669
-52168034 25775234
81311508 29422972
100155752 16614006
39716169 31435959
14389928 17683004
102512577 5357773
-59698952 82186092
13227718 488502
35786182 30451480
-87382823 78444764
18388183 -5412993
37772221 30979550
-97115207 41957634
-94235213 32608880
-74848365 85317838
32638785 29475211
-45584890 40714546
89412560 25896632
-62934538 16940779
-73492939 15323869
-85768006 79947227
-97188548 42359041
71522100 31951319
102813494 8277561
30039938 28524899
-80997367 17645982
19962265 23091379
91309672 24794606
-87592818 22609316
-48982135 30980535
-94608850 33478840
-95486245 65052659
-96606056 61243852
-88999212 24158123
20058328 -6722301
-86598247 79204297
13134566 646717
11877143 13135404
18983096 -5900045
-57762597 20011885
-48696855 69278554
53753174 -16756450
101834691 3077845
102770527 9304216
102203088 12288907
-94326738 32816728
44237930 32284463
-47507071 34306103
93585964 23273346
16060805 19663328
-72959951 85569839
-67350741 15569696
100586132 516833
-63822271 84260840
81136311 29482643
-46890468 35998346
61594337 -16683988
-97477634 44202859
-47140220 35284811
23629686 25474057
-82821283 82140714
-79985847 17150887
37199631 30833889
-70569474 15206852
47434113 -16277631
-47182500 65672696
-89946758 25332719
82917554 -12407571
-54852651 78227986
-91159763 27019379
-45910373 61516648
-53505728 76748681
-45087896 43397307
27291496 27355077
30702806 -12340994
42822458 -15609797
102184411 4089496
-97526433 44573625
19945849 -6639501
-44602651 52582174
44142637 -15829748
-84258488 19688609
68904407 32384007
-78625206 16579377
-48794907 69481306
-97855208 48462821
73611009 31537700
-97873840 51869695
58767900 -16794502
-91342982 27294472
-46918414 64924331
79239745 30091511
43426128 -15713310
-78964343 84128767
-52397709 25469265
-51204575 27145476
12454509 14506291
-97341952 43275408
28261405 -11349251
47675778 32744768
-64634791 84552542
89121323 26054473
-96690939 39950395
11163267 6006067
50917484 33041698
91235063 -8400794
67167147 -16181143
100993878 1241645
-47428853 66332231
-51568205 74229664
11456690 11805210
-71048565 85643724
45037523 32405274
-97834659 48024521
11879743 3297271
-46069993 62132646
61336482 -16698042
101844097 13337403
17671756 -4791574
-96182970 62841871
33370811 29718461
-50789515 27783331
-51643285 26503403
11328220 11282319
35861600 -14032976
99136212 -1597222
-62088185 17327166
-78753984 84211762
97549036 -3420017
17167214 20767951
-84234348 81169947
-45900930 39361712
29567219 28336492
11268590 5433877
26672962 -10625643
55463163 -16803025
-51609739 26551392
85987670 -11150011
-73365725 85530788
17042901 -4210447
-93826268 69122805
11083120 6570768
-97896271 50299532
41839970 -15430566
55382208 33241354
-94087930 32281100
97730232 -3229917
102150006 12460877
-80302684 17299373
-44646043 53369870
76045053 30974484
11017252 9181076
102377350 11662806
-86572182 79228546
-95582267 36068993
44514898 32327271
-97497872 44353795
91275400 24815736
-62663461 17059971
-48290985 68405102
-87861544 77951890
14617828 -1542071
-61575621 17581869
-50511368 28228613
14075994 17252413
35147933 -13828053
66698145 32678162
-78966410 16712782
57439903 33253269
-49259971 70403555
38961046 31265532
-49973040 29134834
-66264462 15810637
-96961048 41173811
15629295 19193454
-57927670 19885476
38876699 -14806212
66072176 -16310488
68969443 -15934623
61546458 33126407
-88981173 76703872
-51084517 27326830
19597527 -6378472
-89596304 24884844
-45462951 59545669
-84288073 19710526
-45690428 60599859
-44820616 55403442
82886269 28859042
31328091 -12573690
-44962467 56573459
102799159 7591702
50536857 -16573799
96045098 -4863149
13542733 -21750
-77539881 16197915
47368143 32709776
90400704 -8899533
-44965976 56599527
23586464 -9009498
-83458471 81720476
-97678528 45927049
-97542355 56140769
-53314611 76522172
-63551776 84155598
-92147728 28575397
-65633144 84861838
25677709 -10136937
12172036 13879438
-66553670 85100192
97239031 20176106
-62256610 17246915
17053059 20659874
-96311052 62383410
-71855538 15206094
-61586031 17576537
-95456828 35703290
-85496729 80177201
-52438568 25415611
11003476 8960177
99158839 -1568230
-63142958 16852021
-95128447 34794703
73429172 -15136454
-83060857 18854723
-92928561 70893527
-73531041 15328201
44516770 -15887808
-51499955 74131017
99359591 -1306434
-90115995 75285748
-51484506 74108584
-97804047 47491212
99951130 16921137
-96787365 60472142
-60233651 18325822
12262943 2350820
-55463441 22003919
36764161 30719579
-77853445 84539221
23166093 -8764602
-91387169 27361687
-71266899 15196276
82791983 -12454494
68989201 32371434
16609515 -3787905
-77524181 84647852
11571767 4224879
-44933774 44484813
-97055493 41645141
-54161759 77493898
102412245 11522875
35296498 -13871434
35879239 30477679
28289030 -11361259
33520924 -13327341
75926031 -14564398
-71318623 85644270
27872325 27617829
95551008 -5292841
-44653795 47351450
14685205 -1628379
-81321043 83023168
-70546769 85633120
56963709 -16815235
