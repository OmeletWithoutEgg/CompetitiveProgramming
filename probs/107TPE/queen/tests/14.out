-7706284.03 -23025163.28
